blogposts/2020/software-engineer-career-paths.md

```diff
@@ -0,0 +1,85 @@
+
+I’m not saying titles don’t matter. They do, depending on where you work. Having a title starting with Senior or Lead can affect how deeply people consider your input, or what projects and opportunities are available to you. If you are doing the same kind/quality/quantity of work as someone else, of course you will care if you don’t have the same title. But it is still a mistake to treat a string of titles as being identical to your career path.
+
+(One thing I love about working at Sourcegraph is that we don’t have those title prefixes. I “lost” my Senior title when I came here, and I wasn’t sure how I would feel about that. But right away I could see how it put early career engineers on a more equal footing with more experienced engineers, which raises the quality of conversations and designs. That’s definitely worth more to me than a title.)
+
+Every career is unique; there’s no one right way to do this. The trick is to find the path that most resonates with _you_. When you are the most engaged, you will be most effective in expanding your own impact. Here are a few anonymized examples of real software engineers and their career paths.
+
+<div class="text-center">
+  <img src="https://sourcegraphstatic.com/blog/career/Maria.jpg" width="70%" alt="Maria illustration">
+</div>
+
+## Maria, Engineer (Technical Breadth)
+
+Maria moved into tech after first working in another industry, but deciding it wasn’t for her. Her superpower was fearlessly asking questions: she never left a meeting or conversation without knowing everything that was going on. She built her knowledge and skill-set rapidly, first in depth and then in breadth. Her personal performance increased.
+
+By becoming an expert in one codebase, and then in several, Maria became a trusted advisor and mentor. Her impact expanded to other engineers, on and off her team.
+
+Maria’s curiosity uncovered technical needs that were not being met, things that didn’t fall cleanly to any one team to solve. By that point, she had the technical skills to formulate solutions, and the connections and communication skills to get buy-in from stakeholders. In time, she was recognized as the primary technical leader in her group.
+
+In one particular instance, Maria started poking at a feature that was unreliable and impractical to use. It was poorly designed and hadn’t scaled well, but changing it would require the coordination of at least a dozen teams (and their roadmaps!). Many others had run into this broken feature before, but never fixed it because “you just can’t push a change through that many teams.”
+
+Maria showed them how. She led the effort to phase it out and replace it with something much better, something that didn’t feel like it was made for the web of 2010, something customers would use and love. A year later, Maria was providing technical guidance and proofs-of-concept for multiple stages of the project to some 60 engineers, working closely with numerous PMs and EMs.
```

Comment:
recommend just saying "product manager" and "engineering manager" instead of PM and EM